eth_account/account.py

```diff
@@ -106,21 +106,21 @@ def decrypt(keyfile_json, password):
         .. code-block:: python
 
             >>> encrypted = {
-             'address': '5ce9454909639d2d17a3f753ce7d93fa0b9ab12e',
-             'crypto': {'cipher': 'aes-128-ctr',
-              'cipherparams': {'iv': '78f214584844e0b241b433d7c3bb8d5f'},
-              'ciphertext': 'd6dbb56e4f54ba6db2e8dc14df17cb7352fdce03681dd3f90ce4b6c1d5af2c4f',
-              'kdf': 'pbkdf2',
-              'kdfparams': {'c': 1000000,
-               'dklen': 32,
-               'prf': 'hmac-sha256',
-               'salt': '45cf943b4de2c05c2c440ef96af914a2'},
-              'mac': 'f5e1af09df5ded25c96fcf075ada313fb6f79735a914adc8cb02e8ddee7813c3'},
-             'id': 'b812f3f9-78cc-462a-9e89-74418aa27cb0',
-             'version': 3}
+            ... 'address': '5ce9454909639d2d17a3f753ce7d93fa0b9ab12e',
+            ... 'crypto': {'cipher': 'aes-128-ctr',
+            ...  'cipherparams': {'iv': '78f214584844e0b241b433d7c3bb8d5f'},
+            ...  'ciphertext': 'd6dbb56e4f54ba6db2e8dc14df17cb7352fdce03681dd3f90ce4b6c1d5af2c4f',
+            ...  'kdf': 'pbkdf2',
+            ...  'kdfparams': {'c': 1000000,
+            ...   'dklen': 32,
+            ...   'prf': 'hmac-sha256',
+            ...   'salt': '45cf943b4de2c05c2c440ef96af914a2'},
+            ...  'mac': 'f5e1af09df5ded25c96fcf075ada313fb6f79735a914adc8cb02e8ddee7813c3'},
+            ... 'id': 'b812f3f9-78cc-462a-9e89-74418aa27cb0',
+            ... 'version': 3}
 
             >>> import getpass
-            >>> Account.decrypt(encrypted, getpass.getpass())
+            >>> Account.decrypt(encrypted, getpass.getpass()) # doctest: +SKIP
```

Comment:
I had to disable because obviously `getpass()` would ask for terminal input.
We could also give an example with an hardcoded value for the `password` that always produces an expected output.

Reply:
Yeah, maybe worth doing both: demonstrating good practices but skipping the test (as you already did), and adding a version that can be doctest-ed.

Reply:
Yes that could be it. Actually I initially wanted to find the password used to generate this output, but haven't found it so far.
I've created a task for adding a doctest that works https://github.com/ethereum/eth-account/issues/81